sub-packages/bionemo-llm/src/bionemo/llm/model/biobert/lightning.py

```diff
@@ -282,96 +277,22 @@ def get_packed_seq_params(batch: SequenceBatch) -> PackedSeqParams:
     )
 
 
-class BioBertLightningModule(
-    pl.LightningModule, iom.IOMixinWithGettersSetters, nlio.ConnectorMixin, LightningPassthroughPredictionMixin
-):
+class BioBertLightningModule(BionemoLightningModule):
     def __init__(
         self,
-        config: MegatronBioNeMoTrainableModelConfig,
-        # TODO: Add transformer_layer_spec when we update mcore
-        tokenizer: Optional[TokenizerSpec] = None,
-        optimizer: MegatronOptimizerModule = MegatronOptimizerModule(
```

Comment:
Why taking out the default optimizer? this will break a few inference calls for esm2

Reply:
we don't need optimizer when running inference

Reply:
This is me getting rid of the redundant lightning class. See the parent class that we actually use now! It confused me before. I modified forward and nothing changed :)